### DISCUSSION TITLE
Upgrade .NET Core version

### DISCUSSION THREAD

**Discussion by Nanouz** [Dotnet-Boxed/Templates#514] [Q&A]
Hi

I am using your .ASP.NET Core 1.1.2 Boilerplate template and wonder If I can upgrade the .net core version to the latest one?

I can't upgrade this via the NuGet Package Manager because it says its blocked. Is there another way?

I also made a lot of websites with your MVC boilerplate and loved it!

Thx a lot!

**Comment by RehanSaeed** [comment]
> I can't upgrade this via the NuGet Package Manager because it says its blocked. Is there another way?

Not sure how or why that can happen.

> wonder If I can upgrade the .net core version to the latest one?

If you have a project created from an old template. consider creating a project from the latest template and then comparing the differences between the two.

> I also made a lot of websites with your MVC boilerplate and loved it!

👍
